src/cascadia/TerminalSettingsModel/ActionAndArgs.cpp

```diff
@@ -271,6 +261,49 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         return make_self<ActionAndArgs>(action, args);
     }
 
+    Json::Value ActionAndArgs::ToJson(const Model::ActionAndArgs& val)
+    {
+        if (val)
+        {
+            if (!val.Args())
+            {
+                // "command": "copy"
+                const auto shortcutActionIter{ ActionToStringMap.find(val.Action()) };
+                if (shortcutActionIter != ActionToStringMap.end())
+                {
+                    return JsonKey(shortcutActionIter->second);
+                }
+            }
+            else
+            {
+                // "command": { "action": "copy", "singleLine": false }
```

Comment:
1. `"command": { "action": "copy" }`

https://github.com/microsoft/terminal/blob/b506817508da9a4d5cd403a9ce1ba9f738eb65d6/src/cascadia/LocalTests_SettingsModel/SerializationTests.cpp#L240-L243

We _could_ make it return `2` by...
1. check if all of the members of the `IActionArgs` are `std::nullopt`.
2. If true, write the shorthand version

But step `1` of that would be really annoying to implement. It could be a bit less annoying to implement given `JsonConverters`, but I digress.